assignment3/app.js

```diff
@@ -0,0 +1,27 @@
+var http = require('http');
+var util = require('util');
+
+var secondlargest = require('./secondlargestarray');
+
+var frequency=require('./calculateFrequency');
+
+var flatten=require('./flatten_object');
+
+var unflatten=require('./unflatten');
+
+var array=[2,4,3,1,5,6];
```

Comment:
a space after comma is expected when listing array elements. Expected decalaration: `[1, 45, 678, 34, 21]`